README.md

```diff
@@ -74,40 +74,17 @@ The purpose of this "documentation skeleton" project is to provide a simple way
 
 To install the package in your project follow these steps:
 
-1. add the Github registry for the @deven-org scope:
+install the package:
 
-    ```bash
-    npm config set @deven-org:registry=https://npm.pkg.github.com/ --location project
-    ```
-
-2. authenticate towards the Github registry (if you are not already):
-
-    1. create a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
-
-        * we recommend to use the "classic" token interface
-        * minimum scope that needs to be set is `read:packages`
-
-    2. login to the Github registry
-        * with npm
-            ```bash
-            npm login --scope=@deven-org --registry=https://npm.pkg.github.com/
-            ```
-
-        * with yarn
-            ```bash
-            yarn npm login --scope my-scope
-            ```
-
-3. install the package:
-    * with npm
-        ```bash
-        npm install @deven-org/documentation-skeleton --save-dev
-        ```
-
-    * with yarn
-        ```bash
-        yarn add @deven-org/documentation-skeleton --dev
-        ```
+with npm
+  
+```bash
+npm install @deven-org/documentation-skeleton --save-dev
+```
+with yarn
+```bash
+yarn add @deven-org/documentation-skeleton --dev
+```
```

Comment:
It's a bit nit-picky, but we could align this with the other `npm/yarn` commands in this file:

To install the package use one of the following commands.

```bash
npm install @deven-org/documentation-skeleton --save-dev
```

or

```bash
yarn add @deven-org/documentation-skeleton --dev
```

```

Reply:
Its a good and valid point, should stay consistant